starter-code/functions-and-arrays.js

```diff
@@ -105,8 +136,17 @@ var hasDog = doesWordExist(words, "dog");
 console.log(hasDog);
 
 // Counting Repetion
-function howManyTimes (words, word) {
-
+function howManyTimes (array, word) {
+  var newTab =[word];
```

Comment:
This will work fine but `newTab`will always have the same word in it. You could instead use an integer and increment it accordingly; it's simpler.